apps/web-client/src/routes/orders/suppliers/reconcile/[id]/+page.svelte

```diff
@@ -50,10 +50,18 @@
 
 	const form = superForm(defaults(zod(scannerSchema)), {
 		validators: zod(scannerSchema),
-		validationMethod: "submit-only"
+		validationMethod: "submit-only",
+		onUpdated: async ({ form: { data, valid } }) => {
```

Comment:
Check out the change I made in my `fix/customer-order-lines` PR. Not essential, but it does improve the experience